ubyssey/templates/about/index.html

```diff
@@ -0,0 +1,156 @@
+{% load staticfiles %}
+{% load dispatch_tags %}
+<!DOCTYPE html>
+<html>
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
+
+  <title>About - UBC's Official Student Newspaper</title>
+  <meta name="description" content="The Ubyssey, founded in 1918, is the definitive source of news on campus. Published every Tuesday during the school year, The Ubyssey is the largest student paper in Western Canada, employs more than 12 full- and half-time editorial staff throughout the year and is supported by an independent, full-time business office." />
+
+  <link rel="apple-touch-icon" sizes="57x57" href="{% static 'images/favicons/apple-touch-icon-57x57.png' %}">
```

Comment:
Since this part is shared with other pages, what do you think about making a shared component like templates/components/favicons.html that all pages can import with {% include 'components/favicons.html' %}?

Reply:
Yea, that's a good call